Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2716,15 +2735,36 @@ private protected virtual void OnDisable()
         SceneView.duringSceneGui -= OnSceneViewGUI;
 #endif
     }
-#endregion
 
-//----------------------------------------------------------------------------------------------------------------------
-    
-    //[TODO-sin: 2020-12-14] m_assetsFolder only makes sense for MeshSyncServer because we need to assign a folder that
-    //will keep the synced resources as edits are performed on the DCC tool side.
-    //For SceneCachePlayer, m_assetsFolder is needed only when loading the file, so it should be passed as a parameter 
-    
-    [SerializeField] private string  m_assetsFolder = null; //Always starts with "Assets"
+
+
+        public int getNumObservers => m_observers?.Count ?? 0;
```

Comment:
do we need this to be public ?

Reply:
Yes, I'm accessing this in Editor/Scripts/Analytics/MeshSyncObserver.cs:L23

Reply:
Done